test/test_http_proxy.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
+
+# Node entity tests
+class TestHttpProxy < MiniTest::Test
+  def test_get_from_json_v3
+    path = 'http://localhost:8080/api/v1/proxy/namespaces/default/services/image-scanner:5001-tcp/image-scanner/api/scan'
+    stub_request(:post, path)
```

Comment:
test name is test_get_* but the request type in the test is post - can you please explain?